docs/virtual_machines/usb_host_passthrough.md

```diff
@@ -0,0 +1,86 @@
+# USB Host Passthrough
+
+This feature is another case of [Host Devices Assigment](host-devices.md), where we can provide a
+USB devices that is plugged in a Node to the VM running in the same Node.
+
+## Requirements
+
+Cluster admin privilege to edit the KubeVirt CR in order to
+- [Enable feature gate](../operations/activating_feature_gates.md) HostDevices
+- Edit the `permittedHostDevices` configuration to expose node USB devices to the cluster
+
+## Exposing USB Devices
+
+In order to assign USB devices to your VMI, you'll need to expose those devices to the cluster under
+a resource name. For this example, we will use the `kubevirt.io/storage` resource name for the
+device with `vendor: "46f4"` and `product: "0001"` [^1].
+
+
+```yaml
+spec:
+  configuration:
+    permittedHostDevices:
+      usb:
+        - resourceName: kubevirt.io/storage
+          selectors:
+            - vendor: "46f4"
+              product: "0001"
+```
+
+After adding the `usb` configuration under `permittedHostDevices` to KubeVirt CR, the KubeVirt's
+device-plugin will expose this resource name and you can use it in your VMI.
+
+
+## Adding USB to your VM
+
+Now, in the VM, you can add in the `devices` > `hostDevices` the reference to the `resource name`
+and give it a local name, for example:
+
+```yaml
+spec:
+  domain:
+    devices:
+      hostDevices:
+      - deviceName: kubevirt.io/storage
+        name: usb-storage
+```
+
+You can find a working example, which uses QEMU's emulated USB storage, under
+[examples/vmi-usb.yaml](https://github.com/kubevirt/kubevirt/blob/main/examples/vm-usb.yaml).
+
+
+## Bundle of USB devices
+
+You might be interested to redirect more than one USB device to a VMI, for example, a keyboard, a
+mouse and a smartcard device. The KubeVirt CR supports assigning multiple USB devices under the same
+`resource name`, so you could do:
+
+```yaml
+spec:
+  configuration:
+    permittedHostDevices:
+      usb:
+        - resourceName: kubevirt.io/peripherals
+          selectors:
+            - vendor: "045e"
+              product: "07a5"
+            - vendor: "062a"
+              product: "4102"
+            - vendor: "072f"
+              product: "b100"
+```
+
+Adding to the VMI configuration:
```

Comment:
Previously we did this in the VM configuration. Should this also be VM or should the other be VMI?

Reply:
Sorry, this somewhat confuses me.  Not sure what you meant by previously. The configuration is done the [kind: VirtualMachineInstance](https://github.com/kubevirt/kubevirt/blob/main/examples/vmi-usb.yaml#L3)

Reply:
L218, in the first example of adding a USB device, it was added to the VM config, whereas this second example, adding multiple, is done by editing the VMI config. 
If there's a reason why one is done on the VM config and the other on the VMI then I think it's worth a short explanation to clarify why. But it sounds like both should be VMI?

Reply:
Correct, both should be VMI. I'll be fixing this and updating shortly.

Reply:
```diff
-Now, in the VM configuration, you can add the `devices.hostDevices.deviceName` and reference the
+Now, in the VMI configuration, you can add the `devices.hostDevices.deviceName` and reference the
 ```